engine/src/test/java/org/camunda/bpm/engine/test/api/multitenancy/TenantIdProviderTest.java

```diff
@@ -1043,6 +1054,160 @@ public void shouldHaveAccessToFormPropertiesFromTenantIdProvider() {
+
+    TestTenantIdProvider.delegate = new StaticTenantIdTestProvider(TENANT_TWO);
+    ProcessInstance processInstance = engineRule.getRuntimeService().startProcessInstanceByKey(PROCESS_DEFINITION_KEY);
+
+    // when
+    engineRule.getIdentityService().setAuthentication("user", null, Collections.singletonList(TENANT_TWO));
+
+    // then
+    ActivityStatisticsQuery query = engineRule.getManagementService()
+        .createActivityStatisticsQuery(processInstance.getProcessDefinitionId())
+        .includeFailedJobs();
+
+    int instances = query.singleResult().getInstances();
+    assertThat(instances).isEqualTo(1);
+  }
+
+  @Test
+  public void shouldQueryForActivityInstancesIncludeIncidentsGetInstances() {
+    // given
+    testRule.deploy(TASK_PROCESS);
+
+    TestTenantIdProvider.delegate = new StaticTenantIdTestProvider(TENANT_ID);
+    engineRule.getRuntimeService().startProcessInstanceByKey(PROCESS_DEFINITION_KEY);
+
+    TestTenantIdProvider.delegate = new StaticTenantIdTestProvider(TENANT_TWO);
+    ProcessInstance processInstance = engineRule.getRuntimeService().startProcessInstanceByKey(PROCESS_DEFINITION_KEY);
+
+    // when
+    engineRule.getIdentityService().setAuthentication("user", null, Collections.singletonList(TENANT_TWO));
+
+    // then
+    ActivityStatisticsQuery query = engineRule.getManagementService()
+        .createActivityStatisticsQuery(processInstance.getProcessDefinitionId())
+        .includeIncidents();
+
+    int instances = query.singleResult().getInstances();
+    assertThat(instances).isEqualTo(1);
+  }
+
+  @Test
+  public void shouldQueryForActivityInstancesIncludeFailedJobsGetJobs() {
+    // given
+    testRule.deploy(FAILING_PROCESS);
+
+    TestTenantIdProvider.delegate = new StaticTenantIdTestProvider(TENANT_ID);
+    engineRule.getRuntimeService().startProcessInstanceByKey(PROCESS_DEFINITION_KEY);
+
+    TestTenantIdProvider.delegate = new StaticTenantIdTestProvider(TENANT_TWO);
+    ProcessInstance processInstance = engineRule.getRuntimeService().startProcessInstanceByKey(PROCESS_DEFINITION_KEY);
+
+    testRule.executeAvailableJobs();
+
+    // when
+    engineRule.getIdentityService().setAuthentication("user", null, Collections.singletonList(TENANT_TWO));
+
+    // then
+    ActivityStatisticsQuery query = engineRule.getManagementService()
+        .createActivityStatisticsQuery(processInstance.getProcessDefinitionId())
+        .includeFailedJobs();
+
+    int jobs = query.singleResult().getFailedJobs();
+    assertThat(jobs).isEqualTo(1);
+  }
+
+  @Test
+  public void shouldQueryForActivityInstancesIncludeIncidentsForTypeGetIncidents() {
+    // given
+    testRule.deploy(Bpmn.createExecutableProcess(PROCESS_DEFINITION_KEY).startEvent()
+        .serviceTask()
+          .camundaDelegateExpression("org.camunda.bpm.engine.test.api.multitenancy.FailingDelegate")
+          .camundaAsyncBefore()
+        .done());
```

Comment:
Can we reuse the `FAILING_PROCESS` here? If not, can we at least make this model a constant as well and reuse it in `shouldQueryForActivityInstancesIncludeIncidentsGetIncidents`?